### ISSUE TITLE
More rewrites

### ISSUE THREAD

**Issue by samuelgruetter** [scalacenter/scalafix#36]
Happy to see that such rewriting can now be done with scalameta instead of manipulating strings :smiley: 

If you're looking for more rewrites needed to go from Scala to Dotty, you can have a look at the list in the README of the [srewrite tool](https://github.com/samuelgruetter/srewrite) that I wrote some time ago to port scalac tests to dotty.

**Comment by olafurpg** [comment]
Thank you for sharing! I totally forgot to respond, so sorry.

I took the liberty to open issues for each documented rewrite in srewrite, https://github.com/scalacenter/scalafix/issues?q=is%3Aissue+is%3Aopen+label%3Arewrite some issues include diffs from running the srewrite test suite. I'll close this issue in favor of the rewrite specific issues.

The current status is that we're getting the scala.meta semantic api up and running! We made huge progress last week (see https://github.com/scalacenter/scalafix/pull/57/files#r100465911) and are currently dealing with some performance issues (see https://github.com/scalacenter/scalafix/issues/63).
